src/Map/MapRasterizer_P.cpp

```diff
@@ -521,32 +544,78 @@ void OsmAnd::MapRasterizer_P::rasterizePolyline(
     }
     else
     {
+        float hmargin = 0.0f;
+        auto drawShifted = [&, this](float hmargin)
+        {
+            SkPath pathHmargin;
+            if (calcPathByTrajectory(context, points31, pathHmargin, hmargin))
```

Comment:
``` canvas.drawPath(pathHmargin, paint);```
Can't find why this new code introduced (and it was not there before), probably because construction ```drawShifted``` is too hard to understand